packages/vsce/package.json

```diff
@@ -19,10 +19,38 @@
   "contributes": {
     "commands": [
       {
-        "command": "zowe-native-proto-vsce.helloWorld",
-        "title": "Hello World"
+        "command": "zowe-native-proto-vsce.connect",
+        "title": "Zowe-SSH: Connect to Host..."
```

Comment:
The server seems to uploaded successfully, but ...
Is this error message expected?

<img width="252" alt="image" src="https://github.com/user-attachments/assets/a8da7283-8c87-48a9-9922-b8f8a0eb7e30" />

Reply:
The `getLocalStorage` API was added in ZE API v3.1. This error may be happening if you have an older version of ZE installed?